homeassistant/components/unifiprotect/sensor.py

```diff
@@ -527,6 +528,15 @@ def _get_alarm_sound(obj: Sensor) -> str:
         ufp_value="is_smart_detected",
         ufp_event_obj="last_smart_detect_event",
     ),
+    ProtectSensorEventEntityDescription(
+        key="smart_obj_licenseplate",
+        name="License Plate Detected",
+        icon="mdi:car",
+        device_class=DEVICE_CLASS_DETECTION,
```

Comment:
I'm missing a `strings.sensor.json` file for this integration.

https://developers.home-assistant.io/docs/internationalization/core#stringssensorjson

Reply:
There is actually nothing to translate here. The value of this sense is a license plate which is not translatable.

Reply:
Ok, then just remove the custom device class. Thanks!